docs/standard/threading/timers.md

```diff
@@ -14,19 +15,21 @@ author: "rpetrusha"
 ms.author: "ronpet"
 ---
 # Timers
-Timers are lightweight objects that enable you to specify a delegate to be called at a specified time. A thread in the thread pool performs the wait operation.  
+
+The <xref:System.Threading.Timer?displayProperty=nameWithType> class enables you to continuously call a delegate at specified time intervals. You also can use this class to schedule the single call to a delegate in a specified time interval. The delegate is executed each time on a <xref:System.Threading.ThreadPool> thread.
+
+The following example creates a timer that starts after one second (1000 milliseconds) and calls the provided delegate every two seconds. When you create a timer, you can provide the object that will be passed to every delegate call. In the example, such an object is used to count how many times the delegate is called. The timer is stopped when the delegate has been called at least 10 times.
+
+[!code-cpp[System.Threading.Timer#2](../../../samples/snippets/cpp/VS_Snippets_CLR_System/system.Threading.Timer/CPP/source2.cpp#2)]
+[!code-csharp[System.Threading.Timer#2](../../../samples/snippets/csharp/VS_Snippets_CLR_System/system.Threading.Timer/CS/source2.cs#2)]
+[!code-vb[System.Threading.Timer#2](../../../samples/snippets/visualbasic/VS_Snippets_CLR_System/system.Threading.Timer/VB/source2.vb#2)]
+
+For more information and examples, see <xref:System.Threading.Timer?displayProperty=nameWithType>.
+
+Another timer that can be used in a multithreaded environment is <xref:System.Timers.Timer?displayProperty=nameWithType> that by default raises events on a <xref:System.Threading.ThreadPool> thread.
   
- Using the <xref:System.Threading.Timer?displayProperty=nameWithType> class is straightforward. You create a **Timer**, passing a <xref:System.Threading.TimerCallback> delegate to the callback method, an object representing state that will be passed to the callback, an initial raise time, and a time representing the period between callback invocations. To cancel a pending timer, call the **Timer.Dispose** function.  
-  
-> [!NOTE]
->  There are two other timer classes. The <xref:System.Windows.Forms.Timer?displayProperty=nameWithType> class is a control that works with visual designers and is meant to be used in user interface contexts; it raises events on the user interface thread. The <xref:System.Timers.Timer?displayProperty=nameWithType> class derives from <xref:System.ComponentModel.Component>, so it can be used with visual designers; it also raises events, but it raises them on a <xref:System.Threading.ThreadPool> thread. The <xref:System.Threading.Timer?displayProperty=nameWithType> class makes callbacks on a <xref:System.Threading.ThreadPool> thread and does not use the event model at all. It also provides a state object to the callback method, which the other timers do not. It is extremely lightweight.  
```

Comment:
Though not as well done here as it should have been, the note was intended to differentiate between timer types in the .NET Framework, which was an enormous source of confusion and complaint. I think it is still useful to add a note before the first level-2 head:

\> \[!NOTE]
\> Some .NET implementations may include additional \`Timer\` classes. See the \<xref:System.Threading.Timer?displayProperty=nameWithType> class for their differences and usage scenarios.